test/functional/test_framework/util.py

```diff
@@ -426,8 +443,8 @@ def sync_mempools(rpc_connections, *, wait=1, timeout=60, flush_scheduler=True):
         pool = [set(r.getrawmempool()) for r in rpc_connections]
         if pool.count(pool[0]) == len(rpc_connections):
             if flush_scheduler:
-                for r in rpc_connections:
-                    r.syncwithvalidationinterfacequeue()
+                for r in get_nodes_least_version(rpc_connections):
+                        r.syncwithvalidationinterfacequeue()
```

Comment:
I'd rather remove the call to sync_mempools. You can do that by deriving the function that calls it.

Reply:
If I only knew, where am I calling *sync_mempools*. Will have to search a bit.

Reply:
I think, it comes indirectly from *setup_network*. This method also calls *sync_* stuff.

Reply:
You can override setup_network and provide your own impl